ivadomed/visualize.py

```diff
@@ -171,21 +171,27 @@ def save_img(writer, epoch, dataset_type, input_samples, gt_samples, preds, wand
         elif isinstance(input_samples, list):
             input_samples = input_samples[0]
 
-        grid_img = vutils.make_grid(input_samples,
+        # the images fed to make_grid appear to be flipped by default even without any data augmentation, 
+        # possibly due to the orientation of the matrix. Hence, the images are flipped in the last two 
+        # dimensions in order for them to appear correctly in Tensorboard/WandB. For more details, see
+        # https://github.com/ivadomed/ivadomed/issues/1272
```

Comment:
Get to the point of the code right away-- start with the WHAT. And no need to write a novel-- details are already in the issue. 
```suggestion
        # Flipping the last two dimensions of the array to accommodate visual output in Tensorboard/WandB. 
        # https://github.com/ivadomed/ivadomed/issues/1272
```

Reply:
Yes, I'm working on trying to not write so many things and be more direct 😓 
Updated the comment in commit [904b143](https://github.com/ivadomed/ivadomed/pull/1273/commits/904b143e6d5f12b2f4dd5ec41900775803151319)